tests/apps/courses/test_models_organization.py

```diff
@@ -760,3 +762,50 @@ def test_models_organization_get_persons_several_languages(self):
         )
         self.assertEqual(Person.objects.count(), 2)
         self.assertEqual(organization.get_persons().count(), 1)
+
+    def test_models_organization_get_organizations_codes_with_2_orgs(self):
+        """
+        Check if the get_organizations_codes cache returns correct result for 2 different orgs.
+        """
+        org_page_code1 = "ORG_XPTO_1"
+        org_page_code2 = "ORG_XPTO_2"
+        organization1 = OrganizationFactory(should_publish=True, code=org_page_code1)
+        organization2 = OrganizationFactory(should_publish=True, code=org_page_code2)
+        organization_page1 = organization1.extended_object
+        organization_page2 = organization2.extended_object
+        with self.assertNumQueries(2):
+            organization_codes1 = list(
+                Organization.get_organizations_codes(organization_page1, "en")
+            )
+            self.assertListEqual(
+                organization_codes1,
+                [org_page_code1],
+            )
+            organization_codes2 = list(
+                Organization.get_organizations_codes(organization_page2, "en")
+            )
+            self.assertListEqual(
+                organization_codes2,
+                [org_page_code2],
+            )
```

Comment:
Better test the number of queries on each call separately?
```suggestion
        with self.assertNumQueries(2):
            organization_codes1 = list(
                Organization.get_organizations_codes(organization_page1, "en")
            )
        self.assertListEqual(
            organization_codes1,
            [org_page_code1],
        )

        with self.assertNumQueries(0):
            organization_codes2 = list(
                Organization.get_organizations_codes(organization_page2, "en")
            )
        self.assertListEqual(
            organization_codes2,
            [org_page_code2],
        )
```